rules.md

```diff
@@ -3,104 +3,131 @@ layout: default
+[Submit your project for the weekly prizes](submissions.html#weekly-prize-submission)
 
-#### Does our solution need to be open-source?
+### Final Prizes
 
-*   No, not necessarily.
-*   For teams that choose to keep their solution proprietary, judges and organizers reserve the right to scrutinize the source code & other assets for compliance with usage rights.
-*   For all teams, failure to allow judges' and organizers' scrutiny may result in disqualification from prizing of any nature.
+Teams also have until April 30th at 23:59 UTC to submit their projects for consideration for the final prizes.
 
-#### Can I work on a project I've already started before CODEVID-19?
+These are larger prizes awarded to a winner and runner-up in four problem areas, as decided by a team of expert judges. There will also be final popular choice prize awarded at the end of the competition.
 
-That's fine, however only projects that were started **on or after March 16, 00:00 UTC** can be submitted for prizing considerations. See [Submissions](#submission) for more details.
+Judging begins once project submissions close, and we're aiming to complete it within one week.
 
-### Submissions
+[Submit your project for the final prizes](submissions.html#final-prize-submission)
 
-#### General
+### Our Challenge
 
-Read these carefully, otherwise we can't consider your team for prizing!
+We're passionate about improving the quality of life for people during and after the COVID-19 Pandemic by addressing the following problems:
 
-*   **Deliverables**
-    *   A link to your working app (whether that's a FindCollabs link, the website URL, or a TestFlight/Google Play opt-in URL).
-    *   A link to a screenshot we can share with voters.
-    *   Instructions for interacting with your app (if not included in the app itself).
-    *   An email address that judges can use to follow up with the team should they have questions or concerns. This will not be shared with the public, or with voters.
-    *   For open-source projects, we encourage a link to the source code repository (e.g. GitHub, GitLab, Bitbucket, etc.).
-*   Your submission does not need to be complete, but you must be able to demonstrate it.
-*   Teams may submit their projects for multiple weekly considerations; however, new progress must be demonstrated leading up to each submission.
-*   Your solution does not need to be open-source, but it must be available (upon request) for scrutiny by organizers and judges.
-*   In the spirit of hackathons, only apps that were started **on or after March 16, 00:00 UTC** will be considered for prizing.
-*   It bears repeating: organizers and judges reserve the right to disqualify entries and winners who violate the [Code of Conduct](code-of-conduct.html) and [Submission](#submission) rules as outlined on this website.
+1. Supporting Crisis Response
+2. (Mis)Understanding The Pandemic
```

Comment:
My knee-jerk reaction to the "(Mis)" part was confusion.  I think grammatically, "Misunderstanding of the pandemimic" is different from "Misunderstanding the pandemic"; the latter is how it's currently read in my mind.

It might be too wordy, but something along the lines of "(Mis)information about the Pandemic" reads less ambiguously.